docs/source/developer_documentation/rocky.md

```diff
@@ -0,0 +1,173 @@
+## Rocky Design
```

Comment:
This should be a level 1 heading. Two level two headings will be rendered as two separate things by sphinx in the docs.